bigbluebutton-html5/client/compatibility/adjust-videos.js

```diff
@@ -1,7 +1,7 @@
 
 (function() {
```

Comment:
Just a heads up that this is changing the `style` and `class` of some elements. This is not a good idea when using React. We probably need to move all this logic to some sort of layout component and rely less on javascript calculations using flexbox.

Reply:
And is using jQuery :puke: